app/src/main/java/org/greenstand/android/TreeTracker/usecases/GetPlanterAccountData.kt

```diff
@@ -0,0 +1,36 @@
+package org.greenstand.android.TreeTracker.usecases
+
+import org.greenstand.android.TreeTracker.api.RetrofitApi
+import org.greenstand.android.TreeTracker.database.TreeTrackerDAO
+import org.greenstand.android.TreeTracker.database.entity.PlanterAccountEntity
+import timber.log.Timber
+import java.util.*
+
+const val TAG: String = "GetPlanterAccountData"

```

Comment:
This needs be moved into the GetPlanterAccountData class.  Otherwise you'd have multiple top level TAG values in the future.